### PR TITLE
GUI instructions added

### PR DESCRIPTION
**Pull request by rwoll**
@ericthewry I changed the instructions to be all GUI. Please proofread and add clarifications/additional explanations where you see fit. 

Additional screenshots are in the images directory in a subdirectory.

Let me know if you have any questions.

### CODE REVIEW

File: student_guide.md [hunk 1]
```diff
@@ -136,22 +119,20 @@ the links in them!***
...
-
-6. Now, add the EGit plug-in to the workbench by clicking ***Window > Show
+5. Now, add the EGit plug-in to the workbench by clicking ***Window > Show
    View > Other... > Git > Git Repositories***.
+6. Add the Git menu to your toolbar by clicking ***Window > Customize Perspective...
+   > Command Groups Availability (tab)***. Check **Git** and **Git Navigation Actions**.
+   Click ok to finish
 
-At this point, Eclipse should look like [this](/images/env_preview.png).
+At this point, Eclipse should look like [this]().
```
Comment (rwoll) [inline student_guide.md]: @ericthewry This is the line causing the build to fail. There should be a new screenshot in the `unprocessed_screenshots` folder to replace this missing link.

File: student_guide.md [hunk 2]
```diff
@@ -136,22 +119,20 @@ the links in them!***
...
-
-6. Now, add the EGit plug-in to the workbench by clicking ***Window > Show
+5. Now, add the EGit plug-in to the workbench by clicking ***Window > Show
    View > Other... > Git > Git Repositories***.
+6. Add the Git menu to your toolbar by clicking ***Window > Customize Perspective...
+   > Command Groups Availability (tab)***. Check **Git** and **Git Navigation Actions**.
+   Click ok to finish
 
-At this point, Eclipse should look like [this](/images/env_preview.png).
+At this point, Eclipse should look like [this](images/unprocessed_screnshots/Screen Shot 2015-09-21 at 7.49.32 AM.png).
```
Comment (ericthewry) [inline student_guide.md]: @rwoll that didn't work... :/ Ill check again in a bit

**Comment by ericthewry** [comment]
@rwoll This should be good

**Comment by rwoll** [comment]
@ericthewry thanks! the screen shots can live on GH and if needed we can rename and move them to an appropriate location in the docs.

**Pull request merged**